examples/chart/teleport/README.md

```diff
@@ -60,6 +60,19 @@ $ kubectl create configmap ca-certs --from-file=ca.pem
 $ helm upgrade --install teleport ./
 ```
 
+## Downloading the chart from the Gravitational repo
+
+Gravitational hosts this Helm chart at http://charts.gravitational.io - it is updated from `master` every night.
```

Comment:
I would specific that this is a `2.16.9` chart.

Reply:
Despite the fact that I've made a big deal of saying "we need to update our stuff to Helm v3", when I looked into this a bit more, there aren't actually any breaking chart changes between Helm v2 and v3.

Charts in Helm 3 use API format v2 rather than v1, but Helm 3 is still compatible with v1 so I don't think we need to explicitly call out that "this is a Helm 2 chart" for now. We have the tiller version dependency specified in `Chart.yaml` which states that you just need `>=2.8.0`.

(my `helm` command locally is using v3 and it worked fine when I tested the commands out earlier)

Reply:
Thanks for the reply and explanation